### ISSUE TITLE
Silent Installation without Root using Shizuku

### ISSUE THREAD

**Issue by rollingmoai** [NeoApplications/Neo-Store#102]
For example, [SkyDroid](https://skydroid.app/) allows for this (could use it with root as well):

![Shizuku on SkyDroid](https://user-images.githubusercontent.com/75471378/139830827-3e132aa4-78af-4ad2-991b-6b50044d0222.png)

**Comment by pusherplayer** [comment]
You read my mind I was about to request the same.
Honestly it would be a good addition  for people who
don't have root access to use adb through shizuku to install apps

**Comment by 1Michael23** [comment]
> You read my mind I was about to request the same. Honestly it would be a good addition for people who don't have root access to use adb through shizuku to install apps

More than just people without root, Shizuku is great with root too as its a more secure method of handing out permissions to apps.

**Comment by foreverxml** [comment]
This would be really nice. I have not figured out how to get silent install working on Droidify (yet) but SkyDroid works.

**Comment by PandoraHayami** [comment]
Any updates of this?

**Comment by k3b** [comment]
If i understood correctly: Neo-Store is gpl (see https://github.com/NeoApplications/Neo-Store/blob/master/COPYING ).

**Is Shizuku-License compatible with gpl?**

From https://github.com/RikkaApps/Shizuku#License :

> For the project as a whole, it is not free. You are FORBIDDEN to distribute the apk compiled by you (including modified, e.g., 
> rename app name "Shizuku" to something else) to any store (IBNLT Google Play Store, F-Droid, Amazon Appstore etc.).

**Comment by Small-Ku** [comment]
The Shizuku API is MIT.
https://github.com/RikkaApps/Shizuku-API

**Comment by prirai** [comment]
I also recommend that this be added. Many apps from fdroid also use this, like there is Tachiyomi, Insular, Darq, Hail, Termux and these apps benefit freatly by enabling Shizuku api.

**Comment by rollingmoai** [comment]
The closest thing that is Neo-Store can do right now are silent **updates** in Android 12 without needing root (#20)

**Comment by scientiac** [comment]
Shizuku has been a must have since I stopped rooting my phone. So there's no harm in adding the feature. Droidify has it but neo having many features than that of droidify doesn't have it. It doesn't make sense. I want to switch to neo so bad but this thing alone is keeping me with droidify.

**Comment by Offerel** [comment]
At least for me, none of the installer options from Droidify works. I have tested all of them for a while, but absolute no update of an app, was updated automatically.

Can you please tell me what I should choose (in Droidify), to get f-droid app updates installed,  without that I have manually accept the update?

**Comment by prirai** [comment]
@scientiac , @Offerel I use neo-store and have noticed that it suits silent updates. Not sure if it works on Android below 12. Silent updates doesn't work almost half the time though.

**Comment by scientiac** [comment]
@prirai I have android 13 and I don't see any "silent" installs

**Comment by machiav3lli** [comment]
Won't be added as long as Shizuku isn't free.

**Comment by rollingmoai** [comment]
It's _not entirely_ free, but how does that have to do with anything?

**Comment by raniesantos** [comment]
I've been indecisive about which of the two I should use since they started as a single project and look very identical.
I guess this makes the decision easier.

**Droid-ify it is then.**

**Comment by Xynonners** [comment]
isn't shizuku licensed under apache 2.0 with that single forbidden? I wouldn't see how that would make the app nonfree.

**Comment by machiav3lli** [comment]
@Xynonners not really, it's a custom license having restrictions. The one restriction blocking this is the one realted to forbiding public builds and distribution of the app, with that it's not free in any sense, it's just code available.

**Comment by Meow-purr** [comment]
@izzysoft is shizuku flosss (tag `@machiav3lli`)

**Comment by IzzySoft** [comment]
@Meow-purr in a way. The license says so, but the TOS disagrees a little. When I last checked there was something with "thou shalt not distribute our binaries" or so.

> For the project as a whole, it is not free. You are FORBIDDEN to distribute the apk compiled by you (including modified, e.g., rename app name "Shizuku" to something else) to any store (IBNLT Google Play Store, F-Droid, Amazon Appstore etc.).

Now tell me how "it is not free" matches the license specified.

```
   2. Grant of Copyright License. Subject to the terms and conditions of
      this License, each Contributor hereby grants to You a perpetual,
      worldwide, non-exclusive, no-charge, royalty-free, irrevocable
      copyright license to reproduce, prepare Derivative Works of,
      publicly display, publicly perform, sublicense, and distribute the
      Work and such Derivative Works in Source or Object form.
```

To highlight: the right to **reproduce, prepare Derivative Works of, publicly display, publicly perform, sublicense, and distribute the Work and such Derivative Works in Source or Object form** is explicitly granted by Apache-2.0. Forbidding the distribution of self-compiled/modified APKs (aka "Derivative Works") contradicts that. Which is why it's not entirely clear – as either the license voids the TOS, or the TOS the license. The TOS is clearly violating [freedom-2](https://en.wikipedia.org/wiki/Free_and_open-source_software#Four_essential_freedoms_of_Free_Software) (The freedom to redistribute copies; counting starts with freedom-0 here).

Before you ask: I distribute **their** APK, and to make sure explicitly asked their permission. But it's quite dark gray area. In dubio pro reo (IANAL) I decided this way; F-Droid would definitely decide the other way as rules are stricter there.

**Comment by Meow-purr** [comment]
> @Meow-purr in a way. The license says so, but the TOS disagrees a little. When I last checked there was something with "thou shalt not distribute our binaries" or so.
> 
> > For the project as a whole, it is not free. You are FORBIDDEN to distribute the apk compiled by you (including modified, e.g., rename app name "Shizuku" to something else) to any store (IBNLT Google Play Store, F-Droid, Amazon Appstore etc.).
> 
> Now tell me how "it is not free" matches the license specified.
> 
> ```
>    2. Grant of Copyright License. Subject to the terms and conditions of
>       this License, each Contributor hereby grants to You a perpetual,
>       worldwide, non-exclusive, no-charge, royalty-free, irrevocable
>       copyright license to reproduce, prepare Derivative Works of,
>       publicly display, publicly perform, sublicense, and distribute the
>       Work and such Derivative Works in Source or Object form.
> ```
> 
> To highlight: the right to **reproduce, prepare Derivative Works of, publicly display, publicly perform, sublicense, and distribute the Work and such Derivative Works in Source or Object form** is explicitly granted by Apache-2.0. Forbidding the distribution of self-compiled/modified APKs (aka "Derivative Works") contradicts that. Which is why it's not entirely clear – as either the license voids the TOS, or the TOS the license. The TOS is clearly violating [freedom-2](https://en.wikipedia.org/wiki/Free_and_open-source_software#Four_essential_freedoms_of_Free_Software) (The freedom to redistribute copies; counting starts with freedom-0 here).
> 
> Before you ask: I distribute **their** APK, and to make sure explicitly asked their permission. But it's quite dark gray area. In dubio pro reo (IANAL) I decided this way; F-Droid would definitely decide the other way as rules are stricter there.

Any lawer here

**Comment by machiav3lli** [comment]
It doesn't need a lawer to clear the situation, it's clear that the dev(s) has added an exception of the license aka. customized it, blocking the aforementioned freedom-2. I don't see a need to discuss any further, concerning who have rights and what not.. I respect their wish (and understand the concern partialy - although it's always questionable if this added exception would stop the misuse of the code) and decide for this project to avoid supporting of this API.

**Comment by IzzySoft** [comment]
Funny thing is: I've just updated my local clone of fdroiddata from upstream master, and noticed this line:

    create mode 100644 srclibs/shizuku.yml

And now I'm puzzled.

```yaml
RepoType: git
Repo: https://github.com/RikkaApps/Shizuku

Prepare: git -c 'url.https://u:p@github.com/.insteadOf=git@github.com:' submodule update --init
```

Seems like the API is deemed FOSS, as the TOS is just affecting the APK? :thinking:

**Comment by Meow-purr** [comment]
> @Meow-purr in a way. The license says so, but the TOS disagrees a little. When I last checked there was something with "thou shalt not distribute our binaries" or so.
> 
> > For the project as a whole, it is not free. You are FORBIDDEN to distribute the apk compiled by you (including modified, e.g., rename app name "Shizuku" to something else) to any store (IBNLT Google Play Store, F-Droid, Amazon Appstore etc.).
> 
> Now tell me how "it is not free" matches the license specified.
> 
> ```
>    2. Grant of Copyright License. Subject to the terms and conditions of
>       this License, each Contributor hereby grants to You a perpetual,
>       worldwide, non-exclusive, no-charge, royalty-free, irrevocable
>       copyright license to reproduce, prepare Derivative Works of,
>       publicly display, publicly perform, sublicense, and distribute the
>       Work and such Derivative Works in Source or Object form.
> ```
> 
> To highlight: the right to **reproduce, prepare Derivative Works of, publicly display, publicly perform, sublicense, and distribute the Work and such Derivative Works in Source or Object form** is explicitly granted by Apache-2.0. Forbidding the distribution of self-compiled/modified APKs (aka "Derivative Works") contradicts that. Which is why it's not entirely clear – as either the license voids the TOS, or the TOS the license. The TOS is clearly violating [freedom-2](https://en.wikipedia.org/wiki/Free_and_open-source_software#Four_essential_freedoms_of_Free_Software) (The freedom to redistribute copies; counting starts with freedom-0 here).
> 
> Before you ask: I distribute **their** APK, and to make sure explicitly asked their permission. But it's quite dark gray area. In dubio pro reo (IANAL) I decided this way; F-Droid would definitely decide the other way as rules are stricter there.
@IzzySoft 

@IzzySoft could i modify under apche and have that under apche

**Comment by IzzySoft** [comment]
@Meow-purr you cannot simply take code from a developer and change the license. You have to honor the license they put it under. You can **use** e.g. MIT code in a GPL app (as GPL has the stricter requirements), but not vice versa.

That said, the said phrase indeed seems to only affect the app, not the libraries (I've just checked with the team).

**Comment by prirai** [comment]
Silent updates work on Android 12 and above when using neo-store. Not sure if it is working for everyone.

**Comment by Offerel** [comment]
No, silent updates never worked for me with Android 12 or 13. I always have to click on "update all" to confirm a update start. Only when this confirmation is also omitted, it is an automatic silent update.

**Comment by machiav3lli** [comment]
@Offerel but do you have battery optimiztion disabled and auto-sync active?

**Comment by Offerel** [comment]
> @Offerel but do you have battery optimiztion disabled and auto-sync active?

Yes. Battery optimization is disabled and auto-sync is active. I get a notification, that new updates are available. But they never get installed automatically. Only when i open the app and click on "Update all", the updates for all apps are installed.

**Comment by Meow-purr** [comment]
fsf liisence team says

> Thank you for the additional information; that helps a lot.
>
>You are right to be confused. The developer of Shizuku made the
>licensing of this project very unclear (and that is putting it mildly.)
>
>No one part "supersedes" the other; licenses and license notices don't
>form a hierarchy. You would have to abide by all the terms set forth by
>the copyright holder/s in order to be able to further distribute and
>modify the work.
>
>I would recommend avoiding this work until you can get the copyright
>holders to release it under a license which respects your freedom, or at
>least clarify whether it is a free software project or proprietary.

**Comment by machiav3lli** [comment]
@Meow-purr thanks! I think this makes things clear (again) for now!

**Comment by machiav3lli** [comment]
@Offerel and more importantely, you're on +A12 and the apps not being installed target (targetSDK) at least two Android versions earlier than yours?

**Comment by Offerel** [comment]
I have tested some apps again today. I have installed apps with an older version. After installation was finished, i got immediately a notification that some apps have a new version. I don't touched this notification and don't opened NeoStore manually. After some time (don't know exactly how long, maybe some hours), the app was updated.

This was the first time I have seen that. My phone is on Android 13, I don't know which SDK this is or which target SDK is needed to get this working. One which works, is marked with "Target SDK 33". 

Im happy with the current result and that it worked in this case the first time.

**Comment by machiav3lli** [comment]
The limitation of the targetSDK of the to-be-updated apps is a system-side one, which unfortunately can't be overwritten, but luckily must of FOSS apps are following even when slowly…

**Comment by Meow-purr** [comment]
as @vvb2060 [said](https://github.com/RikkaApps/Shizuku/issues/69#issuecomment-1595855915)

> We authorize everyone to use shizuku code (Java and Kotlin) because we believe it is enough for developers. If someone wants to use the package name, icon and app name of shizuku (Gradle, PNG and XML), then this is a counterfeit website and we cannot be responsible for its content.

so its more of trademark than copright

**Comment by Meow-purr** [comment]
as discused [here](https://github.com/RikkaApps/Shizuku/issues/69#issuecomment-1595830447)

> 'd like to point out that the "Forbidden" has no meaning at all for any code that says it's Apache 2.0 licensed. If a file says it's Apache 2.0 licensed, it's Apache 2.0 licensed, including the right to recompile and distribute. This is the case irrespective of the README saying the project as a whole doesn't actually comply with the Apache 2.0 License. If someone were to fork the entire master branch, then delete the README file and any file that isn't licensed by the Apache 2.0 License, they'd end up with a resulting branch that's entirely made of Apache 2.0 Licensed files, which thus explicitly allows redistribution of recompiled binaries.



 the code is under apache, the icon and readme is not

**Comment by PeterKW** [comment]
Does something like this be used? https://github.com/depau/fdroid_shizuku_privileged_extension

**Comment by machiav3lli** [comment]
As already said multiple times, it's about Shizuku's license and not some technical aspects, an implementation isn't an issue at all!

**Comment by IzzySoft** [comment]
@machiav3lli as [stated above](https://github.com/NeoApplications/Neo-Store/issues/102#issuecomment-1575202565), doesn't the license issue only affect the Shizuku app itself but not the library? Of course if your argument is the app would still be needed to make this useful (is that so?), the point is clear.

**Comment by PeterKW** [comment]
Very sorry, I've been trying to learn about licenses but getting a bit confused. Are you saying the code is copy left but the documentation isn't?

Droidify uses Shizuku and looks like it has the same license although presumably there's something im missing?
https://github.com/Droid-ify/client/blob/main/LICENSE

Is it a problem for someone to fork under the same license Neo Store and do what Droidify have done calling the Shizuku protocols?

Sorry, if my confusion is complicated to answer. I've tried looking up about the licensing but got confused with all the information looping round and round

**Comment by IzzySoft** [comment]
> Are you saying the code is copy left but the documentation isn't?

You must differentiate between the Shizuku **APP** (which to my understanding *handles* the actions) and the Shizuku **LIBRARY.** The latter is what is implemented in apps to communicate with the "core" in the Shizuku app. And that library is not disputed. Just the Shizuku app itself is.

So to my knowledge, the library is libre, but it needs to communicate with the Shizuku app. If the Shizuku app is not installed on the same device, the library would have no effect. But 1) IANAL and 2) I didn't really deeply dig into it.

**Comment by machiav3lli** [comment]
@PeterKW it's not a license-issue in the conventional sense of such, it's the choice to not "nudge" the users into using a non-libre framework. Surely there may be individuals that may have another stance and decide to fork NS, but that's the freedom a libre project provides.

@IzzySoft exactly, it's the dependence on Shizuku app. One may argue: "but it'd be just optional", I'd say: options create nudges and discourses.

**Comment by Meow-purr** [comment]
but if you delete he readme and iconit is now libre(the app)

**Comment by mattdale77** [comment]
I would like to add my voice to them request for this. I see the discussion around it but it is supported from 4.2.4 in Aurora Store and this is on F-Droid and we know that F-Droid has very strict requirements so in my opinion there should be no issue in implementing the library within Neo Store

**Comment by machiav3lli** [comment]
@mattdale77 I think you misunderstood the blocker. It's not whether this is an F-Droid blocker or not, it's about Shizuku being not free (freedoms 2 & 3 are infringed).

**Comment by Meow-purr** [comment]
to avoid the nudge what about "intalition via [sui](https://github.com/RikkaApps/Sui)" this has compatibilty with shizku api but is using magisk and is free software(the resanson to use sui is so there arent  10 su shels ruuning at once)

**Comment by machiav3lli** [comment]
@Meow-purr I think you misunderstood what SUI is. SUI requires Magisk/Zygisk, which means the user already have su/root, which NS already supports as an installer.

**Comment by Meow-purr** [comment]
yes but this is a silly way to avoid the nudge to get propriatary sofotware(shizku) while still basicly adding support

**Comment by Meow-purr** [comment]
also its designd so each app dosent open a su shell waisting cores

**Comment by PeterKW** [comment]
Droidify can use Shizuku for FDroid installer
https://f-droid.org/packages/com.looker.droidify

**Comment by CyanChanges** [comment]
You're facing to its API, not the implementation.
You can implement an non-free implementation. And every free-software use this API must remove it? Because of your call an implementation may not free enough? 

And you're not using pure-AOSP right? 
You are using your manufacturer's OS. And they may not open source at all, and you must call Android API in your Android App, which must call theirs code somehow.

**Comment by CompeyDev** [comment]
At least, would it be possible to provide a setting to select the installer app? Then, in that case one could use a package installer like [pi](https://apt.izzysoft.de/fdroid/index/apk/dev.sanmer.pi) in order to use shizuku or root as an installation method.

**Comment by machiav3lli** [comment]
@CompeyDev Root is supported since ever and Muntashir's App Manager installer is supported on 1.0.0 (still in beta)

**Comment by CompeyDev** [comment]
The whole point of shizuku is to avoid root access, so I don't think that's what people want.

Never heard of the latter will give the build a test run and lyk.

**Comment by CompeyDev** [comment]
Any CI builds I can get my hands on? Or would I need to manually compile it?

**Comment by machiav3lli** [comment]
Test builds are mainly available on the community channels on Telegram and Matrix.

On the shizuku thingy: I don't think that repeating the obvious/said helps anyone further...

**Comment by machiav3lli** [comment]
> Even if our free-developer is using pure-AOSP, there are binary vendor firmwares being non-free. Not to mention the AArch64 ISA and SoCs from Quaxxmm, Mexxxxek, Saxxxng, Apxxe, etc. Is GitHub ever free? It must be the great sacrifice for our free-developer to stand on this free-world and live on the free-energy. F.

Maybe your poetic self can find sometime to fork (and maintain) the project and add the support it wishes.

**Comment by xiaoxiyao** [comment]
Considering [Dhizuku](https://github.com/iamr0s/Dhizuku)?

**Comment by machiav3lli** [comment]
@xiaoxiyao Already had the discussion in the community groups and with other devs. It's just a thin layer over using Shizuku.

**Comment by Brandon98SE** [comment]
Is there a way to do this without Shizuku on Android 15? I want silent background updates to my applications like with Droid-ify and Google Play.

**Comment by prirai** [comment]
> Is there a way to do this without Shizuku on Android 15? I want silent background updates to my applications like with Droid-ify and Google Play.

Android from 13.0 already allows for silent uodation if the app you're trying to install was updated or installed before from the same store. So it will be taken care of automatically. However first installation of any app will be manual.

**Comment by 0x4c756e61** [comment]
Any progress here? I don't understand why everyone is debating on this since the issue is with the Shizuku app, which neo-store doesn't care about, it only needs the API, which is foss.

Furthermore, if it was really an issue, droid-ify wouldn't be using it either, and shizuku would be dead since no one would be able to use it.

**Comment by CompeyDev** [comment]
I'll quote the reason that app manager's maintainer gave for https://github.com/MuntashirAkon/AppManager/issues/55:

> RikkaApps doesn't have a license for Shizuku and the related issue (https://github.com/RikkaApps/Shizuku/issues/56) is closed without any comment. They've also declined to publish their app on F-Droid (https://github.com/RikkaApps/Shizuku/issues/69) which they can't because the app doesn't have any license (Even if I want, I can't even fork the app to provide a release on F-Droid for the same reason). They do have a license for their API but, I have to say, the whole thing is a bit shady and being a strong supported of freedom, I'm not going to support Shizuku.

**Comment by CyanChanges** [comment]
> I'll quote the reason that app manager's maintainer gave for [MuntashirAkon/AppManager#55](https://github.com/MuntashirAkon/AppManager/issues/55):
> 
> > RikkaApps doesn't have a license for Shizuku and the related issue ([RikkaApps/Shizuku#56](https://github.com/RikkaApps/Shizuku/issues/56)) is closed without any comment. They've also declined to publish their app on F-Droid ([RikkaApps/Shizuku#69](https://github.com/RikkaApps/Shizuku/issues/69)) which they can't because the app doesn't have any license (Even if I want, I can't even fork the app to provide a release on F-Droid for the same reason). They do have a license for their API but, I have to say, the whole thing is a bit shady and being a strong supported of freedom, I'm not going to support Shizuku.

Well, to being a strong support of freedom. You should stop using your phone. Almost every Android device have a close-source modded Android OS. Even if you unlock bootloader and flash Lineage or something, you can't get rid of the bootloader and the hardware though.

**Comment by 0x4c756e61** [comment]
I might look like the advocate of the devil, but afaik the Shizuku app is under MIT licence, with applied restrictions, restrictions applied for the artwork of course, and for the release of the app specifically on stores. And I think being forbidden from releasing a fork on the stores is a good thing, it sure is an annoyance, but Shizuku is a pretty critical app, and having a bad actor publish it on stores without any ways to remove it wouldn't be a fun time.

Nyow I agree the way the issue was handled isn't ideal, but that doesn't make the whole project bad or shady? 

I think the current way the app is managed is a pretty good compromise.

And furthermore, your computer runs proprietary code, the cpu microcode, and you're on a Microsoft ran platform, GitHub, which isn't open source at all, you can't just self-host GitHub. So I think you're all stretching things a little too far, in my own opinion

**Comment by CompeyDev** [comment]
> Well, to being a strong support of freedom. You should stop using your phone. Almost every Android device have a close-source modded Android OS. Even if you unlock bootloader and flash Lineage or something, you can't get rid of the bootloader and the hardware though.

You are strawmanning my argument. Furthermore, your statement isn't even true. There are plenty of open AOSP based ROMs that a lot of people use over OEM ones.

**Comment by 0x4c756e61** [comment]
> > Well, to being a strong support of freedom. You should stop using your phone. Almost every Android device have a close-source modded Android OS. Even if you unlock bootloader and flash Lineage or something, you can't get rid of the bootloader and the hardware though.
> 
> You are strawmanning my argument. Furthermore, your statement isn't even true. There are plenty of open AOSP based ROMs that a lot of people use over OEM ones.

Yeah you are right, a lot of people uses AOSP based ROMs over the OEM ones, but that's nyot what Cyan said. They (correct me if I am using the wrong pronouns) said that the ROMs weren't everything, and that your device also ships a bootloader, which could be proprietary. To wich I add that the hardware could also require some proprietary drivers

Finally none of this has anything to do with neo-store including or not a free and open source API released under the MIT

**Comment by prirai** [comment]
I believe the way the app is currently fine, and if the devs don't want to use shizuku it's fine. If you use shizuku you need to manually enable it on every reboot using wadb. However if you're rooted you can just use the root installer option. On an unrooted device device you can simply go with app manager or the default option as you can then be sure of what app you're installing and what the changelog is (if you're using app manager). Also, once and app is installed, the future updates are silent, so there isn't any dialog during updation. This is true for Android 13+.

**Comment by CompeyDev** [comment]
What benefit exactly do we get from Shizuku support? Neo store already supports app manager installation which already supports wadb installations, for privileged tasks. 

Silent installs aren't really much of a concern either on Android versions >13.

**Comment by 0x4c756e61** [comment]
> What benefit exactly do we get from Shizuku support? Neo store already supports app manager installation which already supports wadb installations, for privileged tasks.
> 
> Silent installs aren't really much of a concern either on Android versions >13.

We get more compatibility with already established installations, what I mean is that most people doesn't know about AppManager, secondly I assume most people using it don't know they can set it up to use adb (at least, I didn't unless I looked it up and configured it myself). Furthermore, shizuku is way more known and featured on a lot of android videos, just has a much broader audience and is more likely to be already installed on the user's device.

Lastly, the shizuku api provides much more than the possibility to (un)install apps, which could come in handy in the future

I really don't see any negatives about adding the shizuku api to neo-store

**Comment by Adhjie** [comment]
Just use adb method like MuntashirAkon AppMamager with different port. AppManager wireless setup is a bit hard to do because it's timed. The wired version is fine.
Shizuku advantage is only convenience really. It's wireless setup could be clicked from phone using dev mode and USB debug wireless.

Other important reason why it's not included in AppManager is actually because of the possibility of making the code too complex for maintainer with adb already implemented, shizuku there would be convenient but hard to maintain.

I'm pretty fine, because I could just use Obtainium, I do like searching directly in f-droid and because this app has settings backup unlike f-droid basic.
Or I could indeed just put most of the repos' links to a note...

Well, there's also the other opposite  decision of X-plore implementing shizuku but not ADB.

Files also works to access SAF storage for the FOSS alt.
I do like how Aves shipped google and free variant eg osm, etc. of its gallery app with the map implementation choices.
But if the license is not clear, I'm fine with the devs, maintainers decision, after all I can't maintain a fork side by side either. Could only write bug and request, can't do pull request.

I just wanted to mention some examples.

**Comment by VoxelPrismatic** [comment]
> What benefit exactly do we get from Shizuku support? Neo store already supports app manager installation which already supports wadb installations, for privileged tasks.

It can install apps in the background, like Obtanium. I don't want to go through every app to hit the "update" button when adb can do it for me.

**Comment by rollingmoai** [comment]
The Shizuku installer option released in https://github.com/NeoApplications/Neo-Store/releases/tag/1.1.3 does not work for me and opens the default package installer.

- See https://github.com/NeoApplications/Neo-Store/issues/788

Also, please remove the "wontfix" label in this issue 😅
